nav2_navfn_planner/include/nav2_navfn_planner/navfn_planner.hpp

```diff
@@ -166,15 +166,6 @@ class NavfnPlanner : public nav2_core::GlobalPlanner
    */
   bool worldToMap(double wx, double wy, unsigned int & mx, unsigned int & my);
```

Comment:
We may as well replace this function while we're at it!

Reply:
Should I keep the RCLCPP_ERROR log? 
Or does the implementation always assume successful conversion from world coordinates to map?